src/components/AboutDialog.tsx

```diff
@@ -10,11 +10,23 @@ import { faCopyright } from '@fortawesome/free-solid-svg-icons';
 import KIPR_LOGO_BLACK from '../assets/KIPR-Logo-Black-Text-Clear-Large.png';
```

Comment:
(ignore location of this comment)

Documentation window didn't get any `tr()`s. Specifically seeing "Documentation" window title and "Search..." in search box